Assets/Plugins/Android/AndroidManifest.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="${applicationId}">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" xmlns:tools="http://schemas.android.com/tools" package="${applicationId}">
```

Comment:
Unity bug! Just needed to add the tools so when the developer specifies the non-default SDK requirement of 19 it works.